examples/tutorials.jl

```diff
@@ -22,6 +22,28 @@
         scene = lines(x, y, color = :blue)
     end
 
+    @cell "Tutorial plot attributes" [tutorial, line, attributes] begin
+        x = 1:10
+        y = rand(10, 2)
+        scene = lines(x, y, linewidth = 2)
+    end
+
+    @cell "Tutorial subplots" [tutorial, line, scatter] begin
```

Comment:
This isn't subplots...more like series.  Subplots is layouting - `vbox(sc1, sc2)`, ...